apps/vs-code-designer/src/app/commands/deploy/deploy.ts

```diff
@@ -288,19 +313,27 @@ async function getProjectPathToDeploy(
   const parametersJson = await getParametersJson(workspaceFolderPath);
   let connectionsData: ConnectionsData;
   let parametizedConnections: ConnectionsData;
-
   const targetAppSettings = await node.getApplicationSettings(identityWizardContext as IDeployContext);
-  const resolutionService = new ResolutionService(parametersJson, targetAppSettings);
+
   try {
     parametizedConnections = JSON.parse(connectionsJson);
+
+    if (parametizedConnections.managedApiConnections && Object.keys(parametizedConnections.managedApiConnections).length) {
+      for (const referenceKey of Object.keys(parametizedConnections.managedApiConnections)) {
```

Comment:
are there any unit tests for this?

Reply:
I did not see any unit tets for the other commands so I did not include any.